### PR TITLE
feat: show geo location in accepted detail

### PR DESCRIPTION
**Pull request by c25vdw**
<details><summary>entering</summary>

image
</details>

<details><summary>click on the button</summary>

image
</details>

### CODE REVIEW

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/AcceptedDetailActivity.java [hunk 1]
```diff
@@ -46,50 +50,85 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
+            Address addr = addresses.get(0);
+            // https://stackoverflow.com/a/19927013/7358099
+            StringBuilder addressString = new StringBuilder("");
+            for (int i = 0; i <= addr.getMaxAddressLineIndex(); i++) {
+                addressString.append(addr.getAddressLine(i)).append("\n");
+            }
+            return addressString.toString();
+        } catch (IOException e) {
+            e.printStackTrace();
+            return "Oops, we could not fetch the address.";
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/AcceptedDetailActivity.java:93]: some strings here need to use string resource

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/AcceptedDetailActivity.java [hunk 2]
```diff
@@ -46,50 +50,85 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
-                                    .setNegativeButton(android.R.string.cancel, null)
-                                    .setIcon(android.R.drawable.ic_dialog_map)
-                                    .show();
-                        },
-                        null
-                ));
+        if (request.getMeetingLocation() != null) {
+            buttonModels.add(
+                    new DetailButtonModel(
+                            "View meetup location",
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/AcceptedDetailActivity.java:108]: one more here

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/MeetLocationDialog.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    public MeetLocationDialog(GeoPoint point) {
+        this.geoPoint = point;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
+        Dialog dialog = super.onCreateDialog(savedInstanceState);
+        dialog.setTitle("View Meet Location");
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/MeetLocationDialog.java:36]: one more here, and it looks like the dialog don't have a title? I'm not sure about this

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/MeetLocationDialog.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        dialog.setContentView(R.layout.map_dialog);
+        dialog.findViewById(R.id.map_search_layout).setVisibility(View.GONE);
+        dialog.findViewById(R.id.map_ic_gps).setVisibility(View.GONE);
+        dialog.findViewById(R.id.map_search_confirm).setVisibility(View.GONE);
+
+        cancelButton = dialog.findViewById(R.id.map_search_cancel);
+        cancelButton.setText("Back");
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/MeetLocationDialog.java:43]: last one here :)

**Review by zhpeng811 — APPROVED** [review]
lgtm, some string hardcoding to fix

**Review by QuocTrungTran — APPROVED** [review]
lgtm

**Pull request merged**